### DISCUSSION TITLE
Contribute/Contact us footer in design system

### DISCUSSION THREAD

**Discussion by Sarah-Lombardi** [UKHomeOffice/design-system#562] [General]
There are inconsistent footers on some design system screens asking the user to contribute or feedback. Some are in blue boxes, some are in the text. 

The information is also out of date in some areas - eg out of date emails.

<img width="872" alt="Screenshot 2024-02-21 at 13 24 40" src="https://github.com/UKHomeOffice/design-system/assets/39190312/a9d4d30f-e5d2-4e3c-8bc5-4fd30791f918">
<img width="943" alt="Screenshot 2024-02-21 at 13 25 34" src="https://github.com/UKHomeOffice/design-system/assets/39190312/0b8287ee-9126-4c81-b305-dae9917c477d">
<img width="896" alt="Screenshot 2024-02-21 at 13 25 51" src="https://github.com/UKHomeOffice/design-system/assets/39190312/6e076ca2-2fc6-4b1f-85ae-dbf4f266c6a0">

**Comment by Sarraounia** [comment]
In the short term to allow for ease of contact (there is currently no email address), the [inclusive language guidance](https://design.homeoffice.gov.uk/accessibility/inclusive-language) feedback footer will be updated to:

![Screenshot 2024-02-21 at 15 49 45](https://github.com/UKHomeOffice/design-system/assets/43995815/b70be2ee-fcb2-40e8-98bc-24fbe3cc0c95)

I will be reviewing all feedback footers across the design system to ensure consistency and will share suggestions.
